2022-06-13 14:30:38,815 INFO    MainThread:20806 [wandb_setup.py:_flush():76] Loading settings from /home/dim/.config/wandb/settings
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_setup.py:_flush():76] Loading settings from /home/dim/depth_2022/wandb/settings
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'main.py', 'program': 'main.py'}
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_init.py:_log_setup():428] Logging user logs to /home/dim/depth_2022/wandb/run-20220613_143038-3riuc01h/logs/debug.log
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_init.py:_log_setup():429] Logging internal logs to /home/dim/depth_2022/wandb/run-20220613_143038-3riuc01h/logs/debug-internal.log
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_init.py:init():462] calling init triggers
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_init.py:init():466] wandb.init called with sweep_config: {}
config: {}
2022-06-13 14:30:38,816 INFO    MainThread:20806 [wandb_init.py:init():515] starting backend
2022-06-13 14:30:38,816 INFO    MainThread:20806 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-13 14:30:38,822 INFO    MainThread:20806 [backend.py:ensure_launched():217] starting backend process...
2022-06-13 14:30:38,825 INFO    MainThread:20806 [backend.py:ensure_launched():222] started backend process with pid: 20838
2022-06-13 14:30:38,826 INFO    MainThread:20806 [wandb_init.py:init():525] backend started and connected
2022-06-13 14:30:38,829 INFO    MainThread:20806 [wandb_init.py:init():596] updated telemetry
2022-06-13 14:30:38,830 INFO    MainThread:20806 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-06-13 14:30:39,578 WARNING MainThread:20806 [wandb_init.py:init():1020] interrupted
Traceback (most recent call last):
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    run = wi.init()
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 631, in init
    run, timeout=self.settings.init_timeout
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 223, in communicate_run
    return self._communicate_run(run, timeout=timeout)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 324, in _communicate_run
    resp = self._communicate(req, timeout=timeout)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/home/dim/anaconda3/envs/pytorchenv/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
